frontend/data.json

```diff
@@ -239,6 +245,7 @@
     "Nelson": { "url": "mailto:nelsonr9608@nauta.cu" },
     "D@vid": { "url": "mailto:david@testrun.org" },
     "merlinux": { "url": "mailto:delta@merlinux.eu" },
+    "missytake": { "url": "mailto:missytake@systemli.org" },
```

Comment:
if you can provide an invite link that is better so people can write encrypted to you from chatmail accounts, currently the field is not exposed in the UI, so it is not super important